cmd/zed/zed.d/statechange-led.sh

```diff
@@ -1,9 +1,14 @@
 #!/bin/sh
 #
-# Turn off/on vdevs' enclosure fault LEDs when their pool's state changes.
+# Turn off/on vdevs' enclosure LEDs when their pool's state changes.
 #
-# Turn a vdev's fault LED on if it becomes FAULTED, DEGRADED or UNAVAIL.
-# Turn its LED off when it's back ONLINE again.
+# Turn fault LED:
+#   ON  if vdev FAULTED, DEGRADED or UNAVAIL.
+#   OFF if vdev ONLINE
```

Comment:
might be best to add an explicit "IGNORE all other states" here, too

Reply:
In a271946 - but superseeded by 138c8d2